declarative-schedule-doc.md

```diff
@@ -339,6 +340,90 @@ scenarios:
 And on top of that you can use aliases to avoid duplication. For more details, check output
 [official documentation of openQA](http://open.qa/docs/)
 
+### 5. Use YAML_SCHEDULE_DEFAULT and YAML_SCHEDULE_FLOWS to YAML schedule
+
+
+We have 3 variables to help YAML scheduler to avoid conditional schedule and repetition. They are YAML_SCHEDULE_DEFAULT, YAML_SCHEDULE_FLOWS, YAML_SCHEDULE.
+
+Please be noted that conditional schedule does not work with the new scheduler that uses DEFAULT and FLOWS.
+
+YAML_SCHEDULE_DEFAULT should point to a "default" minimal schedule that has sections.  You can find them in schedule/yast/sle/flows/default.yaml. Sections can have "normal yaml schedule lines" or are just "empty sockets" (indicated by "[]").
```

Comment:
we could remove "minimal", default is just default, when you tweak the system a bit and change the default you could make it minimal or you could make it a big bigger.  We could add some additional test module to those defaults for convenience:
s/"default"/"default"(product defaults + testing defaults)/

Reply:
those are not sections, they are keys in a yaml file and each key always has a list of test modules to be scheduled, could you please rephrase that. Instead of empty sockets, could just say that is a empty list which act as a hook where you can have more granularity when overwriting.